backend/services/PullRequest.js

```diff
@@ -17,24 +17,25 @@ class PullRequest {
    */
   async getAll () {
     try {
-      let response = await axios.get(pEndpoint(this.startDate, this.endDate))
+      const response = await axios.get(pEndpoint(this.startDate, this.endDate))
 
       this.data = response.data
 
       return this
     } catch (error) {
-      console.error(error)
+      return error
+      // console.error(error)
```

Comment:
Let's use `debug` module here.

```js
const debug = require('debug')(services:pull-request)

// and then...

debug(error)
```

Must also install the module: `npm install --save debug`